src/forklift/plugin.py

```diff
@@ -13,21 +13,20 @@
 
 class ScheduledUpdateBase(object):
 
-    def __init__(self):
-        #: the logging module to keep track of the plugin
-        self.log = logging.getLogger(settings.LOGGER)
-        #: the duration in hours that marks a plugin as dirty
-        self.expires_in_hours = 24
-        #: the parent path to where the source data is
-        self.source_directory = 'C:\\MapData\\'
-        #: the path to where the data in dependencies resides
-        self.source_location = 'SGID10.sde'
-        #: the parent path to where the output will be created
-        self.output_directory = 'C:\\MapData\\'
-        #: the file geodatabase where data will be inserted
-        self.output_gdb_name = 'SGID10.gdb'
-        #: the table names for all dependent data for an application
-        self.dependencies = []
+    #: the duration in hours that marks a plugin as dirty
+    expires_in_hours = 24
+    #: the table names for all dependent data for an application
+    dependencies = []
```

Comment:
You are correct that these arrays would be shared across instances. However, we will only ever have one instance of each of these classes. Since we are defining new classes that override class variables from the base class then they behave as expected. I propose that we keep it this way. I'll add some tests to verify.